src/test/java/tutoraid/model/lesson/PriceTest.java

```diff
@@ -0,0 +1,104 @@
+package tutoraid.model.lesson;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static tutoraid.model.lesson.Price.isValidPrice;
+
+import org.junit.jupiter.api.Test;
+
+class PriceTest {
```

Comment:
I was wondering if we need a case to explicitly tests if `isValidPrice` returns `false` when given a negative number? (e.g. -1.01, -2)